gef.py

```diff
@@ -6045,12 +6049,12 @@ def do_invoke(self, _: List[str], **kwargs: Any) -> None:
 
         dbg(f"[remote] initializing remote session with {gef.session.remote.target} under {gef.session.remote.root}")
         if not gef.session.remote.connect(args.pid):
+            gef.session.remote_initializing = False
             raise EnvironmentError(f"Cannot connect to remote target {gef.session.remote.target}")
         if not gef.session.remote.setup():
+            gef.session.remote_initializing = False
```

Comment:
Same
```suggestion
            gef.session.remote_initializing = False
            gef.session.remote = None
```

Reply:
Might want to write a `reset_remote()` method or something in session